OpenSim/Simulation/Model/CoordinateLimitForce.h

```diff
@@ -235,8 +235,8 @@ OpenSim_DECLARE_CONCRETE_OBJECT(CoordinateLimitForce, Force);
     // damping to constant values beyond the limits. These are heap allocated
     // and owned here, but we don't want these functions to be copied if a
     // CoordinateLimitForce is copied.
-    SimTK::NullOnCopyUniquePtr<SimTK::Function::Step> _upStep;
-    SimTK::NullOnCopyUniquePtr<SimTK::Function::Step> _loStep;
+    SimTK::ResetOnCopy<std::unique_ptr<SimTK::Function::Step> > _upStep;
+    SimTK::ResetOnCopy<std::unique_ptr<SimTK::Function::Step> > _loStep;
```

Comment:
Yuck -- why does swig get to vote on syntax for data members? Do we have to support such an old swig?